src/Plugin/RulesAction/UserRoleAdd.php

```diff
@@ -40,11 +41,14 @@ class UserRoleAdd extends RulesActionBase {
   protected $saveLater = FALSE;
 
   /**
-   * {@inheritdoc}
+   * Assign role to a user.
+   *
+   * @param \Drupal\user\UserInterface $account
+   *   User object.
+   * @param array $roles
```

Comment:
This should use the "\Drupal\user\RoleInterface[]" data type instead of "array".